packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposeOffchainConfig.ts

```diff
@@ -110,24 +110,26 @@ const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInpu
 }
 
 const beforeExecute: BeforeExecute<CommandInput, ContractInput> = (context) => async () => {
+  // Config in contract
   const event = await getLatestOCRConfigEvent(context.provider, context.contract)
-  const offchainConfig = event?.offchain_config
-    ? await deserializeConfig(Buffer.from(event.offchain_config[0], 'base64'))
+  const offchainConfigInContract = event?.offchain_config
+    ? deserializeConfig(Buffer.from(event.offchain_config[0], 'base64'))
     : ({} as OffchainConfig)
-
-  const contractOffchainConfig = {
-    ...offchainConfig,
-    offchainPublicKeys: offchainConfig.offchainPublicKeys?.map((key) => Buffer.from(key).toString('hex')),
+  const configInContract = longsInObjToNumbers({
+    ...offchainConfigInContract,
+    offchainPublicKeys: offchainConfigInContract.offchainPublicKeys?.map((key) => Buffer.from(key).toString('hex')),
     f: event?.f,
```

Comment:
Hmm, this is missing from the proposed Config, why the discrepancy here?

I think `f` should be part of the reported `OffchainConfig` and can be removed from here. Please double-check.

Reply:
`f` is somewhat unique property because although it's part of `offchain_config`, it's being set in `propose_config` and is part of proposal on it's own. So, by the time `offchain_config` is proposed, 'f' is already set in `propose_config` step. I add `f` here to highlight the existing value of it from the previous step.